raiden_contracts/contract_manager.py

```diff
@@ -103,21 +106,75 @@ def contracts_deployed_path(
     return data_path.joinpath(f'deployment_{"services_" if services else ""}{chain_name}.json')
 
 
+@deprecated(reason='Use get_contract_deployment_info()')
 def get_contracts_deployed(
         chain_id: int,
         version: Optional[str] = None,
         services: bool = False,
 ):
     """Reads the deployment data."""
-    deployment_file_path = contracts_deployed_path(
+    return get_contracts_deployment_info(
         chain_id=chain_id,
         version=version,
-        services=services,
+        module='services' if services else 'raiden',
     )
 
-    try:
-        with deployment_file_path.open() as deployment_file:
-            deployment_data = json.load(deployment_file)
-    except (JSONDecodeError, UnicodeDecodeError, FileNotFoundError) as ex:
-        raise ValueError(f'Cannot load deployment data file: {ex}') from ex
+
+def merge_deployment_data(dict1, dict2):
```

Comment:
Add a docstring explaining that this is not just a simple merging. You only merge the contracts keys, and assert that all other keys are equal.

Reply:
will do in this PR.

Reply:
Done.